bundles/CoreBundle/EventListener/WorkflowManagementListener.php

```diff
@@ -46,39 +46,12 @@ class WorkflowManagementListener implements EventSubscriberInterface
      */
     protected $enabled = true;
 
-    /**
-     * @var Manager
-     */
-    private $workflowManager;
-
-    /**
-     * @var Registry
-     */
-    private $workflowRegistry;
```

Comment:
$workflowRegistry missing in new contructor

Reply:
Yes. It was removed because it is a private property which is never called and therefore I assumed it is an artifact which is not needed anymore. Is there any benefit in adding it?

Reply:
My bad, I didn't check for its usage. It is fine then we can remove it. Please remove the use on line 37

Reply:
@dvesh3 Removed (+ little type fix). Should fit now?

Reply:
yes, thanks!